commons/src/main/java/org/infinispan/commons/marshall/ProtoStreamTypeIds.java

```diff
@@ -0,0 +1,88 @@
+package org.infinispan.commons.marshall;
+
+/**
+ * TypeIds used by protostream in place of FQN message/enum names to reduce payload size.
+ * <p>
+ * ONCE SET VALUES IN THIS CLASS MUST NOT BE CHANGED AS IT WILL BREAK BACKWARDS COMPATIBILITY.
+ * <p>
+ * Values must in the range 0..65535, as this is marked for internal infinispan use by the protostream project.
+ * <p>
+ * TypeIds are written as a variable length uint32, so Ids in the range 0..127 should be prioritised for frequently
+ * marshalled classes.
+ * <p>
+ * If message/enum types are no longer required, the variable should be commented instead of deleted.
+ *
+ * @author Ryan Emerson
+ * @since 10.0
+ */
+public interface ProtoStreamTypeIds {
+
+
+   // 1 byte Ids 0..127 -> Reserved for critical messages used a lot
+   int WRAPPED_BYTE_ARRAY = 0;
```

Comment:
@ryanemerson Could we start here from 1 instead of 0? I'd like to reserve 0 for WrappedMessage, which is the mother of all messages ?

Reply:
@anistor Sure. Does that mean that I need to explicitly set that in Infinispan somewhere? I just assumed that it already had a TypeId set, but I guess I didn't think about it's value